src/components/CustomTableRow/index.tsx

```diff
@@ -69,15 +69,38 @@ const DefaultAttributeNames = [
   'operations_hash',
   'signature'
 ];
+
+const PrimaryKeyList = {
+  blocks: ['hash', 'level'],
```

Comment:
This information should be taken from the metadata call `keyType` with values `UniqueKey` and `PrimaryKey`